packages/contracts/src/positionToEntityKey.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+function positionToEntityKey(uint32 x, uint32 y) pure returns (bytes32) {
+  return keccak256(abi.encode(x, y));
+}
```

Comment:
Can I move it to utils?

Reply:
I am not formatting it explicitly, I think generated files are formatted by VSCode using default Solidity formatter.

I've noticed there is another `positionToEntityKey` in utils already. I've updated it in recent PR